tests/unit/base/test_utils.py

```diff
@@ -118,6 +137,55 @@ def test_random_file_name_with_id(self):
         self.assertEqual(filepath, expected)
 
 
+class TestEncodeData(BaseAPITestClass):
+    def test_encode_data(self):
+        mock_data = ['test_encode_data_on', 'test_encode_data_tw', 'test_encode_data_th']
+        expected = [b'dGVzdF9lbmNvZGVfZGF0YV9vbg', b'dGVzdF9lbmNvZGVfZGF0YV90dw', b'dGVzdF9lbmNvZGVfZGF0YV90aA']
```

Comment:
@nsjcorps is there a better way to find the expected values? hardcoding it might create issues if someone changes the `mock_data`

Reply:
I changed it @Ram81 . It just encodes the mock_data_now